demo/working_with_csv_files.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4bd8b1a1-7378-4329-bc0f-9966b307bf6d",
+   "metadata": {},
+   "source": [
+    "# Working with csv-file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "70664de1-b630-429e-ac28-fee74a3192fa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import openai\n",
+    "from bia_bob import bob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e136ac2f-5fb0-4fba-8641-935da5da6e4d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openai.api_key = os.environ[\"OPENAI_API_KEY\"]"
```

Comment:
This line is technically not necessary, because openai reads the API key from the environment, in case users don't set it. We can keep it here if it's important for the storyline. I just wanted to say, if you drop this line, everything will still work exactly the same way.

Reply:
Oki thanks for the hint, I will remove the line then as it is not really storyline-relevant